bitnami/metallb/Chart.yaml

```diff
@@ -28,4 +28,9 @@ sources:
   - https://github.com/metallb/metallb
   - https://github.com/bitnami/containers/tree/main/bitnami/metallb
   - https://metallb.universe.tf
-version: 4.1.10
+version: 4.1.11
+
+# The optional kubeVersion field can define semver constraints on supported Kubernetes versions.
+# Helm will validate the version constraints when installing the chart and fail if the cluster
+# runs an unsupported Kubernetes version.
+kubeVersion: ">= 1.19.0-0"
```

Comment:
Could you please place the new parameter in alphabetical order (between `keywords` and `maintainers`)?
In the same way, we are not adding that default comment in other cases, see some examples at
- https://github.com/bitnami/charts/blob/main/bitnami/kube-prometheus/Chart.yaml#L28
- https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq-cluster-operator/Chart.yaml#L21

Reply:
Updated